### PR TITLE
Added signal ID to branches.

### PR DESCRIPTION
**Pull request by magnusfeuer**
Bumped version to 2.0.0 Alpha+005.
Created new JSON, CSV, vsi, fidl, and VehicleSignalSpecification.id artifacts (+005 suffix)

Implements issue #95

### CODE REVIEW

File: tools/vspec.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #
 # (C) 2018 Volvo Cars
-# (C) 2016 Jaguar Land Rover
+# (C) 2016 Jaguar Land Rover (makes sexier cars than Volvo)
 #
```
Comment (danielwilms) [inline tools/vspec.py]: haha...wanna check the review process :)) maybe we could remove this one before we merge.

**Comment by klotzbenjamin** [comment]
Another remark @magnusfeuer 
It was already an issue some time ago, but the IDs are a bit surprising to me. It starts at 1178, and skips 1193, 1194 and everything between 1216 and 3421.

They are at least consistent between generated files (as it was the case before).

File: tools/vspec.py (hunk 1 above)
Comment (magnusfeuer) [reply tools/vspec.py]: Dang - you caught it.
Fixed.

**Comment by magnusfeuer** [comment]
Commenting on @klotzbenjamin 
 VehicleSignalSpecification.id is the master map between signals and their ID. Can you point me to an example where there is a mismatch? That may be a bug.
The ID assignment is a bit stochastic since it depends on the traversal order of the signal tree loaded by vspec.py. Please do tell if there is a better way of assigning IDs, and I'll see if I can make it happen.

**Comment by magnusfeuer** [comment]
Are we ok if I merge this PR into master?

**Pull request merged**